src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+      Ball ballOne = Lottery.getRandomBall();
+      Ball ballTwo = Lottery.getRandomBall();
+      Ball ballThree = Lottery.getRandomBall();
```

Comment:
According to the checklist, you should use a loop for creating several objects of the same class. Consider using a loop to create the three `Ball` objects to avoid redundancy and improve code maintainability .